client/src/featureform/exceptions.py

```diff
@@ -0,0 +1,10 @@
+
+class InvalidTrainingSetFeatureComputationMode(Exception):
+    def __init__(self, feature_name, feature_variant, mode='CLIENT_COMPUTED', message=None):
```

Comment:
Can you use the enum you created? I assume it might result in a circular dependency so maybe make a new python file called computation_mode.py which everything imports the consts from

Reply:
I missed this in the last PR, so thanks for pointing it out. I'll update this now.